example/Taskfile.json

```diff
@@ -0,0 +1,11 @@
+{
+  "hello": {
+    "cmds": [
+      "echo \"I am going to write a file named 'output.txt' now.\"",
+      "echo \"hello\" > output.txt"
+    ],
+    "generates": [
+      "output.txt"
+    ]
+  }
+}
```

Comment:
Done. I don't know if these examples really highlight the parts of task that you want them to, so you might consider changing them. But I hope this helps some.